Makefile

```diff
@@ -53,6 +53,8 @@ run_func_65: func_65
 run_func_87: func_87
 	$(W) $(MLDSA87_DIR)/bin/test_mldsa87
 run_func: run_func_44 run_func_65 run_func_87
+run_acvp: acvp
```

Comment:
We we missing targets for `run_acvp_XXX`?

Reply:
they should be removed in the .PHONY list. Will do. 
this is wrong in mlkem-native as well.

Reply:
fixed.